pkgs/development/python-modules/aioresponses/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonAtLeast
+, python
+, pbr
+, aiohttp
+, ddt
+, asynctest
+, pytest_3
+}:
+
+buildPythonPackage rec {
+  pname = "aioresponses";
+  version = "0.6.0";
+  disabled = !(pythonAtLeast "3.5");
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0ii1jiwb8qa2y8cqa1zqn7mjax9l8bpf16k4clv616mxw1l0bvs6";
+  };
+
+  buildInputs = [ pbr aiohttp ddt asynctest pytest_3 ];
+
+  # Skip a test which makes requests to httpbin.org
+  checkPhase = ''
+    ${python.interpreter} -m pytest -k "not test_address_as_instance_of_url_combined_with_pass_through"
+  '';
+
+  meta = {
+    description = "Aioresponses is a helper to mock/fake web requests in python aiohttp package.";
```

Comment:
```suggestion
    description = "A helper to mock/fake web requests in python aiohttp package";
```

> https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes `description`

Reply:
Fixed, but it would be better if there were a way to take this data from pypi.